Control/Monad/Trans/Random/Strict.hs

```diff
@@ -0,0 +1,262 @@
+{-# LANGUAGE FlexibleInstances #-}
+{-# LANGUAGE GeneralizedNewtypeDeriving #-}
+{-# LANGUAGE MultiParamTypeClasses #-}
+{-# LANGUAGE Trustworthy #-}
+{-# LANGUAGE UndecidableInstances #-}
+
+{- |
+Module       :  Control.Monad.Trans.Random.Strict
+Copyright    :  (c) Cale Gibbard 2006-2007,
+                (c) Russell O'Connor, Dan Doel and Remi Turk 2006,
+                (c) Eric Kidd 2007
+License      :  MIT-style (see the file LICENSE)
+
+Maintainer   :  byorgey@gmail.com
+Stability    :  experimental
+Portability  :  non-portable (multi-param classes, functional dependencies, undecidable instances)
+
+Lazy random monads, passing a random number generator through a computation.
```

Comment:
Yes it should.

I've submitted a patch (see: [8648613acb6a90d29c154387e7faf3605447e68a](https://github.com/byorgey/MonadRandom/pull/25/commits/8648613acb6a90d29c154387e7faf3605447e68a)).